cmd/ocm-backplane/cloud/console.go

```diff
@@ -144,11 +144,9 @@ func runConsole(cmd *cobra.Command, argv []string) (err error) {
 		// Check API connection with configured proxy
 		if connErr := backplaneConfiguration.CheckAPIConnection(); connErr != nil {
 			logger.Error("Cannot connect to backplane API URL, check if you need to use a proxy/VPN to access backplane:")
-			logger.Errorf("Error: %v", connErr)
-			logger.Info("To troubleshoot connectivity issues, please run the following command:")
-			logger.Info("ocm-backplane health-check")
+			logger.Errorf(" Error: %v", connErr)
 		}
-		return fmt.Errorf("failed to get cloud console for cluster %v: %w", clusterID, err)
+		return fmt.Errorf(" failed to get cloud console for cluster %v: %w.\n[**NOTE:** To troubleshoot the connectivity issues, please run `ocm-backplane health-check`]", clusterID, err)
```

Comment:
Would be better to remove the brackets?

```
ERRO[0004] Connection Error to Backplane API:           
ERRO[0004] cannot connect to Backplane API URL: Head "https://api.stage.backplane.openshift.com": proxyconnect tcp: dial tcp: lookup squid.corp.redhat.com: no such host.
[**NOTE:** To troubleshoot the connectivity issues, please run `ocm-backplane health-check`] 
```

Reply:
Seems there are built-in colors for different levels, like `ERRO` `WARN`
Not sure if it is possible to add different format for helper message

Reply:
I removed the brackets and updated by using ANSI code to highlight the Note messages in yellow. 
![image](https://github.com/user-attachments/assets/c7c0f173-d4af-4c3a-afad-a2ed0834076d)
![image](https://github.com/user-attachments/assets/b6e9407f-5f6b-43be-8eac-b1bf24d8458a)
Let me know if that's ok and align with our general bp-cli promoting message styles